libp2p/Session.cpp

```diff
@@ -115,18 +126,18 @@ bool Session::readPacket(uint16_t _capId, unsigned _packetType, RLP const& _r)
     }
     catch (std::exception const& _e)
     {
-        cnetlog << "Exception caught in p2p::Session::interpret(): " << _e.what()
-                << ". PacketType: " << _packetType << ". RLP: " << _r;
+        LOG(m_netLoggerError) << "Exception caught in p2p::Session::readPacket(): " << _e.what()
+                              << ". PacketType: " << capabilityPacketTypeToString(_packetType)
+                              << " (" << _packetType << "). RLP: " << _r;
         disconnect(BadProtocol);
         return true;
     }
     return true;
 }
 
-bool Session::interpret(P2pPacketType _t, RLP const& _r)
+bool Session::interpretCapabilityPacket(P2pPacketType _t, RLP const& _r)
```

Comment:
Nit about the name: it actually interprets p2p packet, capability packets are interpreted by capability implementations.

Reply:
Well technically the spec refers to these packets as p2p capability packets 😄 From https://github.com/ethereum/devp2p/blob/master/rlpx.md: 

> The capabilities supported on either side of the connection are exchanged in the Hello message belonging to the 'p2p' capability which is required to be available on all connections.

I had originally named the function `interpretP2pCapabilityPacket` but decided that was a bit long and went with just `interpretCapabilityPacket `after seeing that the capability class implementations had functions named `interpretCapabilityPacket`.

I had also considered simply `interpretP2pPacket` but thought that p2p was a bit ambiguous.

Reply:
To me it seems confusing that method of capabilities and this one are named the same, because they interpret different classes of packets... In our code what this doc calls `p2p capability` is not a `Capability` but internal implementation of `Session`